lib/store/watches.js

```diff
@@ -10,24 +10,19 @@ import typeof store from "./index";
 
 export default class WatchesStore {
   kernel: Kernel;
-  @observable watches: Array<WatchStore> = [];
+  watches: IObservableArray<WatchStore>;
 
   constructor(kernel: Kernel) {
     this.kernel = kernel;
-
+    this.watches = observable([]);
```

Comment:
I did this so that I could use `watches.clear` in remove all. I was under the impression `@observable` would make this an `IObservableArray` anyway, but flow doesnt think so. If anyone could explain the difference I would be 🙇 .